src/routes/health.ts

```diff
@@ -0,0 +1,10 @@
+import express from "express";
+
+const router = express.Router();
+
+router.get("/health", (req, res) => {
+    console.log("Health check endpoint hit");
+    res.status(200).send("Healthy");
+});
+
+export default router;
```

Comment:
Default exports are not good because they make code refactoring harder since the import name can be anything, making it difficult to track usages and maintain consistency. They also make tree shaking more difficult and can impact type safety. Named exports are better as they ensure import names match export names, making the code more maintainable and easier to refactor.
```suggestion
export const router = express.Router();

router.get("/health", (req, res) => {
    console.log("Health check endpoint hit");
    res.status(200).send("Healthy");
});
```